### PR TITLE
Bug 1924843:  Update gogo/protobuf to v1.3.2

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,11 +11,13 @@ require (
 	github.com/onsi/gomega v1.10.1
```
Comment (vpickard) [inline go.mod:11]: I was expecting to see:

`	github.com/gogo/protobuf v1.3.2 // indirect
`

in the require section, which would be added when the following command is ran:

`	go get github.com/gogo/protobuf v1.3.2`

Maybe I am missing something? For reference, here is a PR where I did this for ovn-k8s. Thoughts?

Comment (zshi-redhat) [reply go.mod:11]: @vpickard thanks for the review!
There was a discussion in upstream project about whether to use replace or go get.
The conclusion is either one is fine: https://github.com/k8snetworkplumbingwg/network-resources-injector/pull/77#discussion_r590443071

Comment (zshi-redhat) [reply go.mod:11]: @vpickard do you think we can move forward with current patch?

Comment (vpickard) [reply go.mod:11]: @zshi-redhat yes, apologies for not responding earler. 
/lgtm

**Pull request merged**